node/clients/pingClient.ts

```diff
@@ -0,0 +1,24 @@
+import {CacheType, InstanceOptions, IOClient, IOContext} from '@vtex/api'
+
+export default class PingClient extends IOClient {
+  private readonly baseUrl: string
+  constructor(ctx: IOContext, options?: InstanceOptions) {
+    super(ctx, options)
+
+    this.baseUrl = `http://${this.context.host}/returns`
+  }
+
+  public async ping() {
+    return this.http.get(`${this.baseUrl}/ping`, {cacheable: CacheType.None}).catch(() => Promise.resolve())
+  }
+
+  public static getMiddleware = () => async function ping(ctx: Context, next: () => Promise<any>) {
+    ctx.vtex.logger.info({
+      middleware: 'PING',
+      message: `Pinged`,
```

Comment:
message: 'Pinged',

Use single quotes '' for strings.
Template literals should contain interpolation or newlines.